cmd/src/bin/tikv-ctl.rs

```diff
@@ -1042,6 +1043,12 @@ fn main() {
         .version(version_info.as_ref())
         .long_version(version_info.as_ref())
         .setting(AppSettings::AllowExternalSubcommands)
+        .arg(
+            Arg::with_name("log")
+                .long("log")
+                .takes_value(true)
+                .help("Log file for tikv-ctl")
```

Comment:
```suggestion
                .help("Log file path for tikv-ctl. Only fatal logs will be printed")
```

Reply:
`Fatal` is only for RocksDB logs. For general TiKV log, the level is still `info` by default.